go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/mendersoftware/deployments
+
+go 1.13
+
+require (
+	github.com/ant0ine/go-json-rest v3.3.3-0.20170913041208-ebb33769ae01+incompatible
+	github.com/asaskevich/govalidator v0.0.0-20180720115003-f9ffefc3facf
```

Comment:
Question:
Why is this on such an old sha? And not a released version?

Reply:
believe it or not, but `TestUnmarshalLogMessage` fails with the latest. 
* github.com/asaskevich/govalidator v0.0.0-20200428143746-21a406dcc535 test fails
* github.com/asaskevich/govalidator v0.0.0-20180720115003-f9ffefc3facf test passes
just change in the dep made the test fail, it was not that easy to find.